kcp/kcp-e2e/setup-ws-e2e.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
```

Comment:
Shell scripts do not have execute bit set (at least they didn't when I cloned them).